legacy/intermediate_fw/trezor.c

```diff
@@ -143,7 +142,7 @@ int main(void) {
     // don't erase
     reboot_device();
   } else {
```

Comment:
```suggestion
    // Storage probably contains a seed so leave the firmware intact.
    // Invalidating it would cause the bootloader to wipe the storage before
    // installing the target firmware. User will need to confirm installation.
    reboot_device();
  } else {
    // New device. Invalidate the intermediate firmware so that after reboot
    // the bootloader will install the target firmware without asking for user
    // confirmation.
```